### ISSUE TITLE
[Crew-Anwendung] Assignments sollen nach Zuweisung wieder aufgehoben werden können

### ISSUE THREAD

**Issue by hollesse** [innoq/seawatch_planner#111]
Eine Person die einer Position zugewiesen wurde, soll ohne das Assignment zu löschen wieder entfernt werden können.

**Comment by janmax** [comment]
Stimmt, das geht aktuell noch nicht. Versuche ich noch mit umzusetzen. Allerdings hätte ich auch nichts dagegen den Zwischenstand von #109 zu mergen, da der commit langsam recht groß wird.

**Comment by hollesse** [comment]
geht klar :)
